srcpkgs/redis/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'redis'
 pkgname=redis
-version=6.0.9
+version=7.0.4
 revision=1
 makedepends="jemalloc-devel libatomic-devel"
 checkdepends="tcl-devel procps-ng"
 short_desc="Advanced key-value store"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://redis.io"
-changelog="https://raw.githubusercontent.com/antirez/redis/5.0/00-RELEASENOTES"
+changelog="https://github.com/redis/redis/releases/tag/7.0.4"
```

Comment:
Please keep with <https://raw.githubusercontent.com/redis/redis/${version%.*}/00-RELEASENOTES>, tag link shows only last point relase changes and is a heavy html.